src/Processors/QtiV2/In/Interactions/TextEntryInteractionMapper.php

```diff
@@ -25,6 +47,35 @@ public function getQuestionType()
         return $closetext;
     }
 
+    public function getOption($key, $defaultValue = null) {
```

Comment:
I dont think this method should be `public`. Lets restrict the visibility scope to minimize access to it.

Reply:
Sure.